msu/hooks/ui/screens/tooltip/tooltip_events.nut

```diff
@@ -39,16 +39,22 @@
 		return null;
 	}
 
+	o.onQueryMSUTooltipData <- function( _data )
+	{
+		return ::MSU.System.Tooltips.getTooltip(_data.modId, _data.elementId).getUIData(_data);
+	}
+
 	local general_queryUIElementTooltipData = o.general_queryUIElementTooltipData;
-	o.general_queryUIElementTooltipData = function( _entityId, _elementId, _elementOwner )
+	o.general_queryUIElementTooltipData = function( _entityId, _elementId, _elementOwner ) // mod settings should be converted to use onQueryMSUTooltipData
```

Comment:
Please fix as necessary because I don't understand what is going on here. @Enduriel

Reply:
changes to this have been reverted, and the hook has been marked as deprecated